halo2/Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [package]
 name = "halo2"
-version = "0.1.0-beta.1"
+version = "0.1.0-beta.2"
 authors = [
-    "Sean Bowe <sean@electriccoin.co>",
-    "Ying Tong Lai <yingtong@electriccoin.co>",
-    "Daira Hopwood <daira@electriccoin.co>",
```

Comment:
This change is to reflect the contents of the `halo2` crate as of this release, which is now brand-new and only has lines from me. We can re-update it as new code is introduced after this release.